### PR TITLE
Footer first implementation

### PR DESCRIPTION
**Pull request by ZEDz318**
added a pageFooter slot containing logo-cloud with 4 logo-items

### CODE REVIEW

**Review by flicksolutions — CHANGES_REQUESTED** [review]
The footer becomes way to prominent with this huge height. That's because the text gets the same width as the other items.
I think the logo cloud might only make sense for the two logos (images) and not the text. Can you update the footer so that it doesn't take that much height? Don't use logo cloud for the text. maybe don't even use it at all. Make the text the same size.

**Comment by ZEDz318** [comment]
I updated the footer by removing the logo-cloud entirely and replaced it with a normal grid. 
I handled a small issue with text being a bit too big on medium screens and also refrained from defining a set height as the images are relatively small and the default height seemed optimal.
Let me know what you think

**Comment by pdaengeli** [comment]
May I quickly chime in here?

In the base docs of the cooperation we stated that some kind of disclaimer will be implemented that informs visitor about the provisional character of the application:
image

Would this be something to include in the footer? Or are there better ways that are both clear and not too intrusive?

The text could read along the lines of:

> Die vorliegende Anwendung hat prototypischen Charakter und stellt kein dauerhaftes Angebot dar. Die Reproduktion der dargebotenen Materialien ist ohne ausdrückliche Zusage der bestandeshaltenden Institutionen untersagt.

If you think the footer would be a good place for this and can implement an initial version by Friday, it would be perfect.

**Comment by flicksolutions** [comment]
Thank you for your input, @pdaengeli! What Do you say we use a [toast](https://www.skeleton.dev/utilities/toasts) for this and don't integrate it in the footer? That way for one it sticks out more (because the footer is not fixed anymore it wouldn't be visible from the start) and one can also dismiss it! If you say yes I would open a new issue for that.

File: src/routes/+layout.svelte [hunk 1]
```diff
@@ -72,5 +74,28 @@
...
+				class="flex justify-center"
+				src={FooterImg2}
+				alt="Ulrich Becher in Basel"
+				/>
+			</div>
+			<div class="flex justify-center">
+				<p class="h5 md:h6 lg:h5 text-black">Ein Prototyp des Deutschen Exilarchivs (DEA) und des Schweizerischen Literarurarchivs (SLA) </p>
+			</div>
+			<div class="flex items-center justify-center">
+				<p class="h3 text-black">Impressum</p>
```
Comment (flicksolutions) [inline src/routes/+layout.svelte]:
```suggestion
				<p class="h5 text-black">Impressum</p>
```

**Comment by pdaengeli** [comment]
I'm fine with a toast. Preferably one of these two ways:

* toast initially open, but with a relatively short timeout
* button with meaningful label ("Wichtiger Hinweis zum Status dieser Ansicht"), toast initially closed

Both are not ideal, but still better than dealing with cookies in my view (or is there another good way to store state once the reader clicked the toast away?).

Edit: no need to reply here, we can discuss in the issue or in person tomorrow.

File: src/routes/+layout.svelte [hunk 2]
```diff
@@ -72,5 +74,28 @@
 	</svelte:fragment>
 	<!-- Page Route Content -->
 	<slot />
-	<svelte:fragment slot="footer">Page Footer</svelte:fragment>
+	<svelte:fragment slot="pageFooter">
+		<div class="grid grid-cols-1 md:!grid-cols-4 gap-1 lg:ml-10 lg:mr-10">
+			<div>
```
Comment (flicksolutions) [inline src/routes/+layout.svelte]: why is there a wrapping `<div>` here? If I delete it, the behaviour is the same. Or did I miss something?

File: src/routes/+layout.svelte
Comment (flicksolutions) [inline src/routes/+layout.svelte:1]: I don't like that on mobile viewports the logos are stacked on top of each other.
Can you put them side-by-side on one row?
You could do that in doing 2 grid columns and then have the two texts span 2 columns. you know what I mean?

**Review by flicksolutions — CHANGES_REQUESTED** [review]
We're almost there!
Just some small changes required.

**Pull request closed without merging**